files/cron.daily/chkrootkit

```diff
@@ -0,0 +1,61 @@
+#!/bin/sh
+
+set -e
+
+CHKROOTKIT=/usr/sbin/chkrootkit
+CF=/etc/chkrootkit.conf
+LOG_DIR=/var/log/chkrootkit
+
+if [ ! -x $CHKROOTKIT ]; then
+  exit 0
+fi
+
+if [ -f $CF ]; then
+  . $CF
+fi
+
+TESTS=$(chkrootkit -l 2>&1 | cut -d ' ' -f 3-)
+
+# Remove disabled tests from list
+if [ -z $DISABLED_TESTS ]; then
+  for t in $DISABLED_TESTS; do
```

Comment:
I suggest to simply require the list of disabled tests to be separated by `|`, so we can get rid of this loop and make do with a single `sed` invocation:

    TESTS=$(echo $TESTS | sed "s/\\b($DISABLED_TESTS)\\b//g")

(I did not test this, so not sure I got it right.)